mlflow/server/js/src/components/ExperimentListView.js

```diff
@@ -35,19 +35,19 @@ class ExperimentListView extends Component {
           </div>
           <div className="experiment-list-container" style={{ height: experimentListHeight }}>
             {this.props.experiments.map((e) => {
-
+              const active = parseInt(e.getExperimentId(), 10) === this.props.activeExperimentId;
               let className = "experiment-list-item";
-              if (parseInt(e.getExperimentId(), 10) === this.props.activeExperimentId) {
-                className = `${className} active-experiment-list-item`
-              }
+              if (active)
+                className = `${className} active-experiment-list-item`;
```

Comment:
Add curly braces even around 1-line bodies for clarity. We try to do that elsewhere in the code. We plan to add a JavaScript linter soon that should hopefully catch these.